test/test_rosmaster/CMakeLists.txt

```diff
@@ -5,21 +5,7 @@ project(test_rosmaster)
 find_package(catkin REQUIRED COMPONENTS genmsg rostest std_msgs)
 
 if(CATKIN_ENABLE_TESTING)
-  add_message_files(DIRECTORY msg
-    FILES
-    String.msg
-    Arrays.msg
-    CompositeA.msg CompositeB.msg Composite.msg
-    Embed.msg Floats.msg Simple.msg
-    RosmsgA.msg
-    RosmsgB.msg
-    RosmsgC.msg
-    TestArrays.msg
-    TestHeader.msg
-    TestPrimitives.msg
-    TestString.msg
-    TVals.msg
-  )
+  add_message_files(DIRECTORY msg)
```

Comment:
> remove the file names list to let `add_message_files` automatically pick up all `.msg` files under the msg directory

Globbing (source) files in CMake is not recommended.

Reply:
will change to list all messages